maat-court-data-api/src/main/java/gov/uk/courtdata/iojappeal/controller/IOJAppealController.java

```diff
@@ -114,4 +117,13 @@ public ResponseEntity<IOJAppealDTO> updateIOJAppeal(@Parameter(description = "In
 
         return ResponseEntity.ok(updatedIojAppealDTO);
     }
+
+    @PatchMapping("/rollback/{iojAppealId}")
+    @Operation(description = "Rollback an existing Interest of Justice appeal record")
+    @StandardApiResponseCodes
+    public ResponseEntity<Void> rollbackIOJAppeal(@PathVariable @NotBlank Integer iojAppealId) {
```

Comment:
Similar comment as to Ross' PR too, following what Josh has done recently, I'd maybe suggest in method names / class names / variables that IOJ is spelt as `Ioj` and for string descriptions talking about IOJ `IoJ`.